### PR TITLE
Setup react

### PR DESCRIPTION
**Pull request by Marcraphael12**
In this PR, I'm setting up the react app

- The application should run in the browser without errors.

### CODE REVIEW

File: my-app/src/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (GzimAsani) [inline my-app/src/.eslintrc.json]: - [x] This app should be in the root of your directory

File: my-app/src/.hintrc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (GzimAsani) [inline my-app/src/.hintrc]: - [ ] This linter should be in the root of your project

File: my-app/src/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (GzimAsani) [inline my-app/src/package-lock.json]: - [x] I don't see the purpose of this file. You should have only one package-lock.json in the root of your project
- You have 3 package-lock.json files

File: .github/workflows/linters.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+name: Linters
```
Comment (GzimAsani) [inline .github/workflows/linters.yml:1]: - [x] This file should be in the root directroy

File: README.md [hunk 5]
```diff
@@ -28,8 +28,11 @@ Additional description about the project and its features.
 To get a local copy up and running follow these simple example steps.
```
Comment (GzimAsani) [inline README.md:29]: - [x] Let the users know how can they clone your project as well

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Marcraphael12                              
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Gitflow is used :clap: 

## Required Changes ♻️ 
- Summary: Everything of your project should be inside the `my-app` folder that you created even  linters`.github` folder. Please remove the unnecessary files such as 2 other `package-lock.json` fiiles
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
_Check comments below :arrow_down_small: _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 4 above)
Comment (Marcraphael12) [reply .github/workflows/linters.yml:1]: it's almost inside the root directory

File: README.md (hunk 5 above)
Comment (Marcraphael12) [reply README.md:29]: done

File: my-app/src/.hintrc (hunk 2 above)
Comment (Marcraphael12) [reply my-app/src/.hintrc]: this file is to check linters locally, so it should be in the working directory

File: my-app/src/.eslintrc.json (hunk 1 above)
Comment (Marcraphael12) [reply my-app/src/.eslintrc.json]: this file is to check linters locally, so it should be in the working directory

File: .github/workflows/linters.yml (hunk 4 above)
Comment (Marcraphael12) [reply .github/workflows/linters.yml:1]: the folder my-app is not the root directory, it's inside a repo

File: my-app/src/.eslintrc.json (hunk 1 above)
Comment (DeliceLydia) [reply my-app/src/.eslintrc.json]: - [ ] All linter setup has to be in the root directory as the previous reviewer requested

File: my-app/src/.hintrc (hunk 2 above)
Comment (DeliceLydia) [reply my-app/src/.hintrc]: Address this

File: my-app/src/package-lock.json (hunk 3 above)
Comment (DeliceLydia) [reply my-app/src/package-lock.json]: Move `package.json` and `package-lock.json` in the root directory this is what I mean you don't need `my-app` folder you can structure your project like this
rt
this doesn't have to be inside another folder put them on the root directory and linter setup

File: .github/workflows/linters.yml (hunk 4 above)
Comment (DeliceLydia) [reply .github/workflows/linters.yml:1]: Address this

File: README.md (hunk 5 above)
Comment (DeliceLydia) [reply README.md:29]: resolved this

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Marcraphael12, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the previous review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Marcraphael12 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
## Hi @Marcraphael12 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has already been approved and merged This is because most Github features will be unusable when a PR is merged. KIndly submit another CRR with an open PR. please note that the new PR should show the changes required in a specific project
- You can either create a brand new repository or just revert previous changes and open a new PR.
- We need to see a piece of code so that we can perform a fresh review once again on the project. 🙏 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**